src/main/scala/xiangshan/frontend/icache/WayLookup.scala

```diff
@@ -29,31 +29,30 @@ import xiangshan.frontend.ExceptionType
  *      to save area, we separate those signals from WayLookupEntry and store only once.
  */
 class WayLookupEntry(implicit p: Parameters) extends ICacheBundle {
-  val vSetIdx        : Vec[UInt] = Vec(PortNumber, UInt(idxBits.W))
-  val waymask        : Vec[UInt] = Vec(PortNumber, UInt(nWays.W))
-  val ptag           : Vec[UInt] = Vec(PortNumber, UInt(tagBits.W))
-  val itlb_exception : Vec[UInt] = Vec(PortNumber, UInt(ExceptionType.width.W))
-  val meta_corrupt   : Vec[Bool] = Vec(PortNumber, Bool())
+  val vSetIdx: Vec[UInt]        = Vec(PortNumber, UInt(idxBits.W))
+  val waymask: Vec[UInt]        = Vec(PortNumber, UInt(nWays.W))
+  val ptag: Vec[UInt]           = Vec(PortNumber, UInt(tagBits.W))
+  val itlb_exception: Vec[UInt] = Vec(PortNumber, UInt(ExceptionType.width.W))
+  val meta_corrupt: Vec[Bool]   = Vec(PortNumber, Bool())
```

Comment:
I prefer aligned type annotations, it there any way to do it?

Reply:
I suggest move this conversation to #3061.

Reply:
There does exist one way to align the annotation, but the formatted code will be
```
val vSetIdx:        Vec[UInt] = Vec(PortNumber, UInt(idxBits.W))
val waymask:        Vec[UInt] = Vec(PortNumber, UInt(nWays.W))
val ptag:           Vec[UInt] = Vec(PortNumber, UInt(tagBits.W))
val itlb_exception: Vec[UInt] = Vec(PortNumber, UInt(ExceptionType.width.W))
val meta_corrupt:   Vec[Bool] = Vec(PortNumber, Bool())
```
which looks more natural to me.

However, considering that all the annotations do not have the same length, this might be too much. I prefer
```
val a: Vec[UInt] = Vec(2, UInt(2.W))
val bb: Bool     = Bool()
```
to
```
val a:  Vec[UInt] = Vec(2, UInt(2.W))
val bb: Bool      = Bool()
```

Reply:
Sadly, the frontend team vote 5 to 4 in favor of the more aligned version and the scalafmt is changed accordingly.